framework/src/transfers/MultiAppGeneralFieldUserObjectTransfer.C

```diff
@@ -102,43 +103,26 @@ MultiAppGeneralFieldUserObjectTransfer::evaluateInterpValuesWithUserObjects(
          (!point_found || _search_value_conflicts || _nearest_positions_obj);
          ++i_from)
     {
+      Real distance = 1;
```

Comment:
Why 1?

Reply:
it s a fine number. I can do 0

Reply:
looks like 0 breaks conflict detection

Reply:
So this doesn't impact the outcome of `acceptPointInOriginMesh`?

Reply:
should not. acceptPointInOriginMesh should provide the distance when using nearest-position. It should not use the distance